### PR TITLE
feat(cli): add blockExplorers option on registry init

### PR DESCRIPTION
**Pull request by nbayindirli**
### Description

- adds blockExplorers option on registry init

### Drive-by changes

- none

### Related issues

- none

### Backward compatibility

- yes

### Testing

- manual

### CODE REVIEW

File: typescript/cli/src/config/chain.ts [hunk 1]
```diff
@@ -111,6 +114,42 @@ export async function createChainConfig({
   }
 }
 
+async function addBlockExplorerConfig(metadata: ChainMetadata): Promise<void> {
+  const wantBlockExplorerConfig = await confirm({
+    default: false,
+    message: 'Do you want to add a block explorer config for this chain',
```
Comment (paulbalaji) [inline typescript/cli/src/config/chain.ts:120]: what style are we going for in the cli, should we phrase this as "Do you want to configure a block explorer for this chain"?

File: typescript/cli/src/config/chain.ts [hunk 2]
```diff
@@ -111,6 +114,42 @@ export async function createChainConfig({
...
+    });
+    const family = (await select({
+      message: 'Select the type (family) of block explorer:',
+      choices: Object.entries(ExplorerFamily).map(([_, value]) => ({ value })),
+      pageSize: 10,
+    })) as ExplorerFamily;
+    const apiKey =
+      (await input({
+        message:
+          "Optional: Enter an API key for the explorer, or 'enter' to skip. Please be sure to remove this field if you intend to add your config to the Hyperlane registry:",
```
Comment (paulbalaji) [inline typescript/cli/src/config/chain.ts:140]: just to disambiguate "enters"
```suggestion
          "Optional: Provide an API key for the explorer, or hit 'enter' to skip. Please be sure to remove this field if you intend to add your config to the Hyperlane registry:",
```

**Review by paulbalaji — COMMENTED** [review]
code change lgtm

**Comment by paulbalaji** [comment]
don't forget changelog

File: typescript/cli/src/config/chain.ts (hunk 1 above)
Comment (nbayindirli) [reply typescript/cli/src/config/chain.ts:120]: i'm alright with both, this was to emulate the current format in the other config functions.

i've added you comment here to capture in the future: https://github.com/hyperlane-xyz/hyperlane-monorepo/issues/3770#issuecomment-2231431779

File: typescript/cli/src/config/chain.ts (hunk 2 above)
Comment (nbayindirli) [reply typescript/cli/src/config/chain.ts:140]: done

**Review by paulbalaji — APPROVED** [review]

**Pull request merged**